controllers/cfg/controllers_cfg.go

```diff
@@ -40,18 +41,25 @@ var defaultControllersCfg = &ControllersCfg{
 	BuilderConfigMapName:          "sonataflow-operator-builder-config",
 }
 
+type GAV struct {
+	GroupId    string `yaml:"GroupId,omitempty"`
+	ArtifactId string `yaml:"ArtifactId,omitempty"`
+	Version    string `yaml:"Version,omitempty"`
```

Comment:
```suggestion
	GroupId    string `yaml:"groupId,omitempty"`
	ArtifactId string `yaml:"artifactId,omitempty"`
	Version    string `yaml:"version,omitempty"`
```

For ubiquity with the other properties.